lib/core/src/client/preview/start.js

```diff
@@ -19,6 +20,8 @@ const classes = {
 };
 
 function showMain() {
+  subscriptionsStore.clearUnused();
```

Comment:
Why is it called here and not right after the render()?

Reply:
Mostly to keep a future possibility of async rendering

Reply:
But you will be able to `await render()` and then call to the "clearUnused", what am I missing?

Reply:
OK makes sense

Reply:
My idea was that if `render` didn't call `showMain`, then something went wrong, and it could be the reason for decorator not to be marked as used